### DISCUSSION TITLE
Support password manager autofills (keychain, etc.)

### DISCUSSION THREAD

**Discussion by patrickyan** [zoidsh/meru#352] [Feature Requests]
Currently, you cannot autofill passwords (adding account or re-entering passwords on existing accounts). Keychain and other password managers should be supported.

**Comment by patrickyan** [comment]
@timche So are all the issues/features that were closed added in the latest version, or was it just for a clean reset for the repo?

**Reply by timche** [reply]
Clean reset of the repo. There were no new releases.
